### ISSUE TITLE
safety category information for training set

### ISSUE THREAD

**Issue by youngwanLEE** [DripNowhy/MIS#1]
Hi, I'm very interested and impressed by your work!!

Could you provide a safety category for each training image, as you already provided in the test set?

It would be very helpful if you could share the information with the safety research community.

Thanks in advance.

**Comment by DripNowhy** [comment]
Hi, thank you for your interest!!

We will organize a more detailed JSON file as soon as possible, including categories and subcategories. I will reply to this issue when the data is updated.🤗

**Comment by youngwanLEE** [comment]
@DripNowhy Thanks for the quick reply. 'm eagerly awaiting your good news.

**Comment by DripNowhy** [comment]
@youngwanLEE  Hi. The training JSON file has been updated with category and sub-category, which were classified using GPT-4o based on the original harmful question and harmful objects. You can access the updated version [here](https://huggingface.co/datasets/Tuwhy/MIS_Train).

If you find our work helpful for your project, we’d greatly appreciate it if you could consider giving the repository a star! 😀

**Comment by youngwanLEE** [comment]
@DripNowhy Thanks!!
